src/main/core-api/java/com/mysql/cj/conf/PropertyDefinitions.java

```diff
@@ -713,6 +713,16 @@ public enum DatabaseTerm {
                 new BooleanPropertyDefinition(PropertyKey.useAwsIam, DEFAULT_VALUE_FALSE, RUNTIME_NOT_MODIFIABLE,
                         Messages.getString("ConnectionProperties.useAwsIam"), "0.3.0", CATEGORY_SECURITY, Integer.MAX_VALUE),
 
+                //
+                // CATEGORY_AWS_SECRETS_MANAGER
+                //
+
+                new StringPropertyDefinition(PropertyKey.secretsManagerRegion, DEFAULT_VALUE_NULL_STRING, RUNTIME_MODIFIABLE,
```

Comment:
Taking into account that Secret Manager plug will be relocated to a separate repo, there will be no access to the driver classes. So these new properties should be defined/maintained by plugin code.